draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -117,16 +116,31 @@ wherever possible.
 
 ## Feature Summary
 
-* This specification defines
- - a format for the payload of an Issuer-signed JWT containing selectively disclosable claims that include object properties (name-value pairs), array elements, and nested data structures built from these,
- - a format for data associated with the JWT that enables selectively disclosing those claims,
- - facilities for binding the JWT to a key and associated data to prove possession thereof, and
- - a format, extending the JWS Compact Serialization, for the combined transport of the JWT and associated data that is suitable for both issuance and presentation.
-* An alternate format utilizing the JWS JSON Serialization is also specified.
-* This specification enables combining selectively disclosable claims with
-  clear-text claims that are always disclosed.
-* For selectively disclosable claims that are object properties, both the key and value are always blinded.
-
+This specification defines two primary data formats:
+
+* SD-JWT: A format for signed, selectively-disclosable JSON content, together
+  with a set of selectively disclosed claims
+  - A format for the payload of an Issuer-signed JWT containing selectively
+    disclosable claims that include object properties (name-value pairs), array
+    elements, and nested data structures built from these
+  - A format for data associated with the JWT that enables selectively
+    disclosing those claims
+  - A format extending the JWS Compact Serialization, allowing for the combined
+    transport of the Issuer-signed JWT and the disclosure data for selectively
+    disclosed claims
+  - An alternate format extending the JWS JSON Serialization, also allowing for
+    transport of the Issuer-signed JWT and disclosure data
+
+* SD-JWT-KB: A format for proving posssion of a private key bound to an SD-JWT
+  - A facility for associating a JWT to a key pair
+  - A format for a Key Binding JWT that proves possession of the private key of
+    the associated key pair
+  - A format extending the SD-JWT format for the combind transport fo the SD-JWT
+    and the proof of possession
+
+The payload of an SD-JWT can contain a combination of selectively disclosable
+claims as well as claims that are always disclosed.  For selectively disclosable
+claims that are object properties, both the key and value are always blinded.
```

Comment:
```suggestion
claims as well as claims that are always disclosed.
```
I think there is a way to blind only the object? like address: { _sd: [] } ?

Reply:
That's different.  What you're describing is hiding the fields within the object vs. hiding the field itself.  You're not actually hiding the `address` field, you're hiding fields within that object.  Even if you hide all the fields, you still reveal that it's an object.

There is no syntax here by which you can reveal that there is a field with a given name in an object without revealing anything about the contents of that field.